microsoft/testsuites/core/timesync.py

```diff
@@ -325,12 +325,16 @@ def verify_timesync_ntp(self, node: Node) -> None:
                 "which is provided in the chrony package."
             )
         ntp = node.tools[Ntp]
-        hwclock = node.tools[Hwclock]
+        if not isinstance(node.os, BSD):
+            hwclock = node.tools[Hwclock]
         service = node.tools[Service]
         # 1. Stop systemd-timesyncd if this service exists.
         service.stop_service("systemd-timesyncd")
         # 2. Set rtc clock to system time.
-        hwclock.set_rtc_clock_to_system_time()
+        if not isinstance(node.os, BSD):
+            hwclock.set_rtc_clock_to_system_time()
+        else:
+            node.execute("adjkerntz -i")
```

Comment:
It should be implemented as an alternative to hwclock. You can only implement `set_rtc_clock_to_system_time` method firstly.

Reply:
Should that be in the hwclock.py file then or should it live somewhere else?

Reply:
in the hwclock.py like other BSD tools

Reply:
Added it to Hwclock.py and confirmed it solves the install issue.